public/views/pages/styleguide.liquid

```diff
@@ -1,106 +1,69 @@
 ---
 layout: modules/components/styleguide
+slug: styleguide(/:component_id)
```

Comment:
What do you think about splitting it into `app/views/pages/styleguide/index.liquid` and `app/views/pages/styleguide/show.liquid` ? `component_id` could be mandatory parameter then

Reply:
I just separated and created `sidebar` component.